miqa/core/tests/factories.py

```diff
@@ -10,3 +14,75 @@ class Meta:
     email = factory.Faker('safe_email')
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
+
+
+class SiteFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Site
+
+    id = factory.Faker('uuid4')
+    name = factory.Faker('safe_email')
```

Comment:
The name of a site in practice is usually something like `duke`, `ohsu`, or `ucsd`. I think this would be a better fake:
```suggestion
    name = factory.Faker('word')
```